runtime/convertTypes.go

```diff
@@ -514,9 +546,9 @@ func exportFunctionType(
 		for i, typeParameter := range t.TypeParameters {
 
 			typeBound := typeParameter.TypeBound
-			var convertedParameterTypeBound cadence.Type
+			var convertedParameterTypeBound cadence.TypeBound
 			if typeBound != nil {
-				convertedParameterTypeBound = ExportMeteredType(gauge, typeBound, results)
+				convertedParameterTypeBound = exportTypeBound(gauge, typeBound, results)
```

Comment:
Perhaps I just couldn't find it, but is there an equivalent `importFunctionType` or similar that also needs to change here?